### PR TITLE
'Solution'

### PR DESCRIPTION
**Pull request by Viktor-Beniukh**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,3 +1,49 @@
...
+        return self.km == other
+
+    def __le__(self, other):
+        return self.km <= other
+
+    def __ge__(self, other):
+        return self.km >= other
+
+    def __len__(self):
+        if isinstance(self.km, int):
```
Comment (IvanRamyk) [inline app/main.py:48]: And if it's a float value?

Comment (IvanRamyk) [reply app/main.py:48]: I believe you don't need this if-statement

**Review by IvanRamyk — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 2]
```diff
@@ -1,3 +1,49 @@
...
+        self.km = km
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers"
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        if isinstance(other, int):
```
Comment (Abnormaltype) [inline app/main.py:13]: And if it's a float value?

File: app/main.py [hunk 3]
```diff
@@ -1,3 +1,49 @@
...
+            return Distance(self.km + other)
+        return Distance(self.km + other.km)
+
+    def __iadd__(self, other):
+        if isinstance(other, int):
```
Comment (Abnormaltype) [inline app/main.py:18]: the same here

File: app/main.py [hunk 4]
```diff
@@ -1,3 +1,49 @@
...
+            self.km += other
+            return self
+        self.km += other.km
+        return self
+
+    def __mul__(self, other):
+        if isinstance(other, int):
```
Comment (Abnormaltype) [inline app/main.py:25]: the same here

File: app/main.py [hunk 5]
```diff
@@ -1,3 +1,49 @@
...
+            return Distance(self.km * other)
+
+    def __truediv__(self, other):
+        if isinstance(other, int):
```
Comment (Abnormaltype) [inline app/main.py:29]: the same here

**Review by Abnormaltype — CHANGES_REQUESTED** [review]

**Pull request closed without merging**